src/PiggyBank.java

```diff
@@ -0,0 +1,22 @@
+public class PiggyBank {
+    private int balance;
+
+    public int getBalance() {
+        return balance;
+    }
+
+    public void setBalance(int balance) {
+        this.balance = balance;
+    }
+
+    public void deposit(int amount){
+        balance += amount;
+    }
+
+    public void withdraw(int amount){
+        if(amount <= balance){
+           balance -= amount;
```

Comment:
If amount is greater than balance, then you should log a message saying it is so

Reply:
I talked to management and they said it wasn't worthy.